### PR TITLE
[15.0][ADD] stock_barcodes_picking_batch_revision: New module

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT45432

ping @carlosdauden @CarlosRoca13

### CODE REVIEW

**Comment by rousseldenis** [comment]
@sergio-teruel Still draft ?

**Comment by sergio-teruel** [comment]
@rousseldenis Changed!!

**Review by carlosdauden — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

File: stock_barcodes_picking_batch_revision/models/stock_move_line.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Copyright 2023 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    is_reviewed = fields.Boolean()
```
Comment (pedrobaeza) [inline stock_barcodes_picking_batch_revision/models/stock_move_line.py]: I think this field has a very common name and can collapse with other developments. It should be prefixed by a distinctful name like `barcodes`:

```suggestion
    barcodes_is_reviewed = fields.Boolean()
```

You can add migration scripts for not losing data.

File: stock_barcodes_picking_batch_revision/models/stock_picking_batch.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class StockPickingBatch(models.Model):
+    _inherit = "stock.picking.batch"
+
+    requested_review = fields.Boolean()
```
Comment (pedrobaeza) [inline stock_barcodes_picking_batch_revision/models/stock_picking_batch.py]: The same:

```suggestion
    barcodes_requested_review = fields.Boolean()
```

**Comment by sergio-teruel** [comment]
Changes Done!!!

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**